src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.task.TaskList;
+
+/**
+ * Command to close the application
+ */
+public class ExitCommand extends Command {
+    public ExitCommand() {
+        setCommandString("bye");
+    }
+
+    /**
+     * Exits the application.
+     *
+     * @param input Full user input
+     * @param taskList The list of tasks
+     * @return The response
```

Comment:
Noticed in a few classes but response differs for each classes so I thought maybe it would be good to show what kind of response it will have in the javadocs?